addons/payment_adyen/models/payment_transaction.py

```diff
@@ -218,13 +320,20 @@ def _process_feedback_data(self, data):
         if self.provider != 'adyen':
             return
 
+        # Extract or assume the event code. If none is provided, the feedback data originate from a
+        # direct payment request whose feedback data shares the same payload as an 'AUTHORIZATION'
+        # webhook notification.
+        event_code = data.get('eventCode', 'AUTHORISATION')
+
         # Handle the acquirer reference
-        if 'pspReference' in data:
+        if 'pspReference' in data and event_code not in ['CAPTURE', 'CANCEL']:
```

Comment:
Isn't it `CANCELLATION` eventCode ?
https://docs.adyen.com/development-resources/webhooks/understand-notifications#transaction-events

Reply:
`resolved`